src/index.html

```diff
@@ -11,7 +11,142 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field  form-field__first">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            required
+            minlength="2"
+            maxlength="15"
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            autocomplete="off"
+            required
+            minlength="2"
+            maxlength="10"
+          >
+        </label>
+        <label class="form-field">
+          How old are You?
+          <input type="number"
+            max="100"
+            min="1"
+            value="12"
+          >
+        </label>
+        <div class="form-field">
+          Full date of birth:
+          <input type="date">
+        </div>
+        <div class="form-field">
+          <label for="checkbox">I accept the term of the agreement</label>
+          <input type="checkbox" id="checkbox">
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <label class="form-field  form-field__first">
+          E-mail:
+          <input
+            type="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <lable class="form-field  form-field__first">
+          Do you love cats?
+          <input
+            type="radio"
+            id="love_cats"
+            name="do_you_love_cats?"
+            value="I_love_cats)"
+          >
+          <label for="love_cats">Yes</label>
+          <input
```

Comment:
Краще розіділяти логічні блоки (тут я розділяю input + label для yes і для no)

```suggestion
          <label for="love_cats">Yes</label>
          
          <input
```